components/IngredientSelector/Navigator.tsx

```diff
@@ -147,8 +147,12 @@ export default function Navigator() {
           );
         }}
         unit={getUnit(currentQuantity, quantityValue!)}
+
         title={currentQuantity.quantity_ingredient_name}
+       
         step={currentQuantity.quantity_step}
+
+        quantity_default_weight_per_unit={currentQuantity.quantity_default_weight_per_unit}
```

Comment:
Not very useful but it's a convention to write props in pascal case

I don't see why "default" since the user will never have to modify the weight per unit

```suggestion
        unit={getUnit(currentQuantity, quantityValue!)}
        title={currentQuantity.quantity_ingredient_name}
        step={currentQuantity.quantity_step}
        weightPerUnit={currentQuantity.quantity_default_weight_per_unit}
```